### PR TITLE
Convert Logstash style tags to Forward protocol's tag

### PR DESCRIPTION
**Pull request by cosmo0920**
Hi, I've tried to handle user-defined tags in this codec plugin.
How about start to handle user-defined forward protocol tags?

Regards,

### CODE REVIEW

File: lib/logstash/codecs/fluent.rb [hunk 1]
```diff
@@ -51,6 +53,18 @@ def encode(event)
     @on_event.call(event, MessagePack.pack([tag, epochtime, data.merge(LogStash::Event::TIMESTAMP => event.timestamp.to_iso8601)]))
   end # def encode
 
+  def forwardable_tag(event)
+    tag = event.get("tags") || "log"
+    case tag.class
```
Comment (colinsurprenant) [inline lib/logstash/codecs/fluent.rb]: when using a case statement to test for an object class, `.class` should not be used:
```
case tag
when Array
  ...
when String
 ...
end
```

**Comment by colinsurprenant** [comment]
This looks reasonable - left a comment on the `case` statement.

File: lib/logstash/codecs/fluent.rb (hunk 1 above)
Comment (cosmo0920) [reply lib/logstash/codecs/fluent.rb]: Oops, I’ll fix it.

Comment (cosmo0920) [reply lib/logstash/codecs/fluent.rb]: I've fixed it.

File: lib/logstash/codecs/fluent.rb [hunk 2]
```diff
@@ -51,6 +53,18 @@ def encode(event)
     @on_event.call(event, MessagePack.pack([tag, epochtime, data.merge(LogStash::Event::TIMESTAMP => event.timestamp.to_iso8601)]))
   end # def encode
 
+  def forwardable_tag(event)
+    tag = event.get("tags") || "log"
```
Comment (colinsurprenant) [inline lib/logstash/codecs/fluent.rb]: can a tag be empty in fluentd protocol? in other words, do we need to assign a default "log" tag when there is no tag in the event?

Comment (cosmo0920) [reply lib/logstash/codecs/fluent.rb]: No, `tag` cannot be empty in fluentd protocol.
We need to assign default value for event which has no tag.

**Comment by colinsurprenant** [comment]
LGTM - thanks for your contribution @cosmo0920 ! 
I will go ahead and merge this PR and will bump version and publish.

**Comment by colinsurprenant** [comment]
Created version bump PR #22

@cosmo0920 Forgot to mention: please submit a PR agains the `CONTRIBUTORS` file if you want to be added in it! Thanks.

**Pull request merged**